src/core_modules/capture-core/components/Buttons/SimpleSplitButton.component.js

```diff
@@ -21,7 +22,7 @@ const SimpleSplitButton = (props: Props) => {
                 <Menu>
                     {
                         dropDownItems.map(i => (
-                            <MenuItem label={i.text} value={i.text} onClick={i.onClick} />
+                            <MenuItem key={uuid} label={i.text} value={i.text} onClick={i.onClick} />
```

Comment:
irrelevant to the overall scope of the PR. However I got it fixed sine this line would throw an error in the console.

Reply:
I also prefer the `uuid` package. If you also agree we could go with that from now on. :)

Reply:
yes, I agree. At some point we will get rid of d2-utilizr I think.

Reply:
Yes think so too.